lepton/image.go

```diff
@@ -204,17 +214,26 @@ func addMappedFiles(src string, dest string, m *Manifest) {
 }
 
 func buildImage(c *Config, m *Manifest) error {
+
 	//  prepare manifest file
-	var elfmanifest string
+	elfmanifest := m.String()
 
-	elfmanifest = m.String()
 	// invoke mkfs to create the filesystem ie kernel + elf image
-	createFile(mergedImg)
-	args := []string{}
+	mergedImgPath := filepath.Join(c.LocalTMPDir, mergedImg)
```

Comment:
Why is this necessary? creating and closing the temp file. Any way it's going to be created.

Reply:
I thought that there is no need to keep it open...

Reply:
ah, got what you mean.. I'll remove `mergedImg.Close()`